### ISSUE TITLE
Toggleable broken bones in game settings

### ISSUE THREAD

**Issue by Ventatulty** [studio-minus/ppg-bugs-and-requests#1629]
Feature description
a new mechanic in the game settings to disable completely a character on having broken bones when too much damage on him.

Feature purpose
i wanted so much this feature, cause the only way to restore the broken bone of the character is on the context menu, when you choose to mend bone so i just needed a different way to disable and enable entirely the character of having broken bones on his body, but a toggleable mechanic in the game settings.

Additional context
Nothing to say

**Comment by DindinYT37** [comment]
The mending and life syringe also restore broken bones.

**Comment by dequalityed** [comment]
or just use mending syringe

**Comment by PossiblyRemember** [comment]
could be added in the gore settings

**Comment by Ventatulty** [comment]
> could be added in the gore settings

yeah that's what i'm talking about in game settings a broken bone should be added in gore settings like u said
